recipe/meta.yaml

```diff
@@ -1,54 +1,23 @@
-{% set version = "4.8.7" %}
+{% set version = "5.6.2" %}
 
 package:
   name: qt
   version: {{ version }}
 
 source:
   fn: qt-everywhere-opensource-src-{{ version }}.tar.gz
-  url: http://download.qt-project.org/official_releases/qt/4.8/{{ version }}/qt-everywhere-opensource-src-{{ version }}.tar.gz
-  md5: d990ee66bf7ab0c785589776f35ba6ad
+  url: http://download.qt.io/official_releases/qt/5.6/{{ version }}/single/qt-everywhere-opensource-src-{{ version }}.tar.gz
+  sha256: 7551f5af1312082c3c8ab52f5e102e3f83aae2dd96dc4a3a9a197f3e58b3214c
   patches:
-    # Make Qt accept its open source license without user interaction
-    - acceptance.patch
-
-    # Make the paths in config.cpp longer so they work with conda-build 2.x
-    - config.cpp.longpath.patch  # [unix]
-
-    # Compile qmake with the system architecture. Taken from the MacPorts project:
-    # https://trac.macports.org/browser/trunk/dports/aqua/qt4-mac/files/patch-configure.diff
-    - qmake-arch.patch  # [osx]
-
-    # Set the minimum deployment target to 10.9
-    - osx_deployment_target_10.9_qmake.conf.patch  # [osx]
-
-    # Enabling our test to set the rpath by making Qt aware
-    # of how to do this on OSX
-    - osx_rpath.qmake.conf.patch  # [osx]
-
-    # fix use of deprecated OSX API. Taken from Homebrew:
-    # https://github.com/Homebrew/legacy-homebrew/issues/40585
-    - qpaintengine_mac.patch  # [osx]
-
-    # Make Qt work with Ubuntu modifications to scrollbars for Gtk
-    - ubuntu_disable_scrollbars.patch  # [linux]
-
-    # Build QtWebkit with gcc-6, on platform linux-g++
-    - qtwebkit-gcc-6.patch  # [linux]
-
-    # Disable warning about Windows 10 not being a supported platform by Qt
-    - disable_win10_warning.patch  # [win]
-
-    # On Windows, for VS 2015 (Python 3.5+), rather drastic modifications are
-    # necessary because Microsoft finally got around to defining some standard
-    # functions. There's the vs_2015_int.patch file that does this. The changes in this patch were
-    # found in forums, and originated at (Author appears to be Peter Zhigalov):
-    # https://fami.codefreak.ru/gitlab/peter/qt4/commit/45e8f4eef3923e03c6939d0c17170980685857ef.diff
-    - vs2015_int_fix.patch  # [win and py >= 35]
```

Comment:
Well... in a way this is clearer as you can see the common options in one place and the different options in another. But I am +0 to this one. I just find it easier to maintain this way as it helps to keep the options in sync.